### ISSUE TITLE
mwc-select fails chrome a11y check for aria attributes

### ISSUE THREAD

**Issue by e111077** [material-components/material-components-web-components#1237]
**Describe the bug**

Chrome lighthouse a11y audit fails on select with the following error:

`[aria-*] attributes do not match their roles`

**To Reproduce**
1. Open select demo
2. run accessibility audit in chrome

**Expected behavior**

no audit failure

**Comment by e111077** [comment]
We've decided to move onto a spec change proposal as Voice Over will not announce on role="listbox". Also required will not announce on either NVDA or Voice Over unless the listbox is expanded. This means tabbing over a select will not announce "required" when in a form.

**Comment by e111077** [comment]
Related: w3c/aria#1253
